Source/WebKit/Shared/skia/WebCoreArgumentCodersSkia.cpp

```diff
@@ -32,9 +32,41 @@
 #include "CoreIPCSkData.h"
 #include "StreamConnectionEncoder.h"
 #include <WebCore/Font.h>
+#include <WebCore/FontCache.h>
+#include <WebCore/FontCustomPlatformData.h>
 
 namespace IPC {
 
+void ArgumentCoder<SkString>::encode(Encoder& encoder, const SkString& string)
```

Comment:
Please use serialization.in files instead of this.

Reply:
We need more documents to migrate. It took me hours to find out the undocumented "wrapped by" keyword.

Reply:
`wrapped by` is only for ObjC type. I can't use it for Skia types now. Needs to modify generate-serializers.py. Will try in a follow up patch.

Reply:
I'd like to write down like:
~~~
SkString wrapped by CoreIPCSkString
SkFontStyle::Slant wrapped by uint8_t
~~~